packages/api-clients/open-api/eserviceTemplateApi.yml

```diff
@@ -1328,6 +1328,9 @@ paths:
       responses:
         "200":
           description: EService template version attributes updated
+          headers:
+            X-Metadata-Version:
+              $ref: "#/components/headers/MetadataVersionHeader"
```

Comment:
Revert all unrelated changes after base branch change! We should see only changes to purpose-process